include/PathOrderOptimizer.h

```diff
@@ -784,17 +784,20 @@ class PathOrderOptimizer
         best_candidate_finder.appendCriteriaPass(main_criteria_pass);
 
         // ########## Step 2: add fallback passes for criteria with very similar scores (e.g. corner on a cylinder)
-        const AABB path_bounding_box(points);
+        if (path.seam_config_.type_ == EZSeamType::SHARPEST_CORNER)
```

Comment:
I wonder if it's better to single out the other two instead of locking it to sharpest corner only. It does work fine in the current situation however.

Reply:
I assume that for the other options, seam alignment is not something the user is looking for, so just skipping it somehow makes sense. Could be debated for "shortest".